doc/modules/mod_event_pusher_http.md

```diff
@@ -33,48 +33,56 @@ It must be defined in the [`outgoing_pools` settings](../advanced-configuration/
 
 ## Options
 
-* `pool_name`: name of the pool to use (as defined in outgoing_pools)
-* `path`: path part of an URL to which a request should be sent (will be appended to the pool's prefix path).
-* `callback_module`: name of a module which should be used to check whether a notification should be sent.
+#### `modules.mod_event_pusher_http.pool_name`
+* **Syntax:** non-empty string
+* **Default:** `""`
```

Comment:
`"http_pool"`? (it is an atom in the code but this would be equivalent looking from the TOML perspective, right?)
https://github.com/esl/MongooseIM/blob/3e0cd59c7cffd53ec7c709b525fcdec8026870e4/src/event_pusher/mod_event_pusher_http.erl#L45